src/components/TopicCard.jsx

```diff
@@ -0,0 +1,15 @@
+import { Link } from "react-router-dom"
+
+function TopicCard({topic, slug}) {
+
+    return (
+        <Link className="link" to={`/topics/${slug}/articles`} topic={topic} slug={slug}>
```

Comment:
completely your choice but I think /topics/${slug} is fine without /articles on the end